cobbler/grub.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import netaddr
+
+
+def parse_grub_remote_file(file_location):
+    prot = ""
+    if file_location.startswith("ftp://"):
+        logging.warning("ftp protocol not supported by grub. Only http and tftp [%s]" % file_location)
```

Comment:
We do not want to use the standard logger at this point. We have our own logger defined in cobbler. This one then uses the logging class.

Reply:
I thought the idea is to use the python logging facility on longterm.

commit d19c1af39618ee5b53f8d8681b19dd42e780d16a
    switch from clogger to python logger

clogger is only a wrapper nowadays.
Re-writing all logger/logging code parts would be a huge effort/patch, but in theory all logging should change to  plain logging.*() calls I thought.